CodeWars Day 2/solution2.js

```diff
@@ -0,0 +1,4 @@
+function fixTheMeerkat(arr) {
+  var meerkats = arr;
```

Comment:
There is no need to assign arr to another variable, if you wanted you can just return arr.reverse();